content/boilerplates/notes/1.2.2.md

```diff
@@ -0,0 +1,4 @@
+## Bug fixes
+
+- Fix crash in Istio's JWT Envoy filter caused by malformed JWT ([Issue 15084](https://github.com/istio/istio/issues/15084))
```

Comment:
this one is still open.  i commented in it

Reply:
I'll fix the issue commenting.